metricbeat/module/docker/helper.go

```diff
@@ -52,24 +51,15 @@ func ExtractContainerName(names []string) string {
 	return strings.Trim(output, "/")
 }
 
-func BuildLabelArray(labels map[string]string) []common.MapStr {
+func BuildLabelArray(labels map[string]string) common.MapStr {
```

Comment:
@andrewkroh Did as suggested. Now let's wait the CI tasks complete :smile:

Hope this can help!